todo/src/util/tooltipManager.js

```diff
@@ -0,0 +1,189 @@
+      this.registerElementInfo(elementInfo.element, elementInfo);
+    });
+  },
+
+  edit(elementSelector, elementInfo) {
+    if (!(elementSelector in store.tooltipInfos)) {
+      throw new Error('정의되지 않은 요소를 수정할 수 없습니다.');
+    }
+    this.registerElementInfo(elementSelector, elementInfo, true);
+  },
+
+  add(elementSelector, elementInfo) {
+    this.registerElementInfo(elementSelector, elementInfo);
+  },
+
+  remove(elementSelector) {
+    if (!(store.tooltipInfos[elementSelector])) {
+      return;
+    }
+    document.querySelectorAll(elementSelector).forEach(element => {
+      element.removeEventListener('mouseover', store.events[elementSelector][element]);
+    });
+    store.events[elementSelector] = {};
+    store.tooltipInfos[elementSelector] = {};
+  },
+
+  registerElementInfo(elementSelector, elementInfo, isEdit = false) {
+    if (!elementSelector) {
+      throw new Error('selector를 정확하게 입력해 주십시오.');
+    }
+
+    const contents = elementInfo.contents ?? '';
+    let delay = elementInfo.delay ?? 0;
+    if (Number.isNaN(delay)) {
+      delay = 0;
+    }
+    store.tooltipInfos[elementSelector] = {
+      contents,
+      delay
+    };
+
+    if (!isEdit) {
+      this.addMouseOverEvent(elementSelector);
+    }
+  },
+
+  addTooltip(target, contents) {
+    const tooltipContainer = document.createElement('div');
+    tooltipContainer.className = 'tooltip-container';
+
+    let tooltipText = target.querySelector('.tooltip');
+    if (tooltipText !== null) {
+      return tooltipText;
+    }
+    tooltipText = document.createElement('div');
+    tooltipText.appendChild(document.createTextNode(contents));
+    tooltipText.className = 'tooltip';
+
+    tooltipContainer.appendChild(tooltipText);
+    target.insertBefore(tooltipContainer, target.firstChild);
+
+    store.targets.push(target);
+
+    return tooltipText;
+  },
+
+  addMouseOverEvent(elementSelector) {
+    const elements = document.querySelectorAll(elementSelector);
+    elements.forEach(element => {
+      const mouseoverEvent = this.createMouseOverEvent(element, elementSelector);
+      store.events[elementSelector] = store.events[elementSelector] || {};
+      store.events[elementSelector][element] = mouseoverEvent;
+      element.addEventListener('mouseover', mouseoverEvent);
+    });
+  },
+
+  createMouseOverEvent(element, elementSelector) {
+    return () => {
+      setTimeout(() => {
+        this.addTooltip(element, store.tooltipInfos[elementSelector].contents);
+      }, store.tooltipInfos[elementSelector].delay);
+    };
+  },
+
+  addMouseMoveEvent() {
+    window.addEventListener('mousemove', event => {
+      store.targets.forEach(target => {
+        if (!(this.isMouseInTarget(event, target))) {
+          this.removeTooltip(target);
+        }
+      });
+    });
+  },
+
+  isMouseInTarget(event, target) {
+    const targetRect = target.getBoundingClientRect();
+
+    const tooltip = target.querySelector('.tooltip');
+    if (tooltip === null) {
```

Comment:
```suggestion
    if (!tooltip) {
```
이건 falsy value를 쓰면 더 간단하겠어요